microreactors/gcmr/assembly/RIA_steady_state/BISON.i

```diff
@@ -0,0 +1,464 @@
+TsInit = 1150.0 # Solid initial temperature
```

Comment:
I expect the MOOSE input files in `RIA_transient`, `RIA_steady_state` and `Flow_blockage_transient` are mostly identical. (how different are RIA_steady-state to Flow-blockage steady-state?) 
If input are different, we can keep the current organization. If some of these files are identical, then I recommend merging these 2/3 folders and just have different files names for those that needs duplication. I recommend removing any input that is duplicate.

Reply:
There are some differences between the MOOSE inputs in the RIA_transient and RIA_steady_state folders.  The Griffin model in the RIA and Flow blockage simulations have some differences in terms of the control rod modeling, also transient BISON models have some differences where no perturbation of the coolant speed was considered in the RIA simulation.

Reply:
why are there two steady states folders though ?
steady_state and RIA_steady_state

Reply:
The initial control rod position at steady state considered in the two simulations are different, so the Griffin inputs at steady state are a little different.